Various test failures when running locally

CI is green, but I have local failures, possibly due to some variance in my elastic setup?

```
............................................................................................................................................................................................................................................................EEE..........................2025-07-23 11:30:55,146 arches.app.search.search ERROR    2025-07-23 11:30:55.146562: WARNING: search failed for query: {'bool': {'should': [{'match_phrase_prefix': {'value': {'query': 'alpha gamma delta'}}}, {'match_phrase_prefix': {'value.folded': {'query': 'alpha gamma delta'}}}, {'match': {'value.folded': {'query': 'alpha gamma delta', 'fuzziness': 'AUTO', 'prefix_length': 3}}}], 'must': [{'term': {'language': 'en'}}], 'must_not': [], 'filter': [{'terms': {'nodegroupid': ['307c4274-c8c1-11ed-a172-0242ac130009', '402c95f2-c8c1-11ed-87a7-0242ac130009', '4780cbb2-c8c0-11ed-a172-0242ac130009', '20000000-0000-0000-0000-100000000000', '6584b9c6-c8bf-11ed-a644-0242ac130009', '96f14fa6-c8bf-11ed-a644-0242ac130009', 'aa5a71de-c8c0-11ed-a172-0242ac130009', 'b74e29fe-c8c4-11ed-aab4-0242ac130009', 'be25bdf0-c8bf-11ed-a172-0242ac130009', 'bf611c24-c8be-11ed-b6a9-0242ac130009', 'd697b522-c8c0-11ed-87a7-0242ac130009', 'c9b3828e-17b3-11eb-a708-acde48001122', 'c9b38568-17b3-11eb-a708-acde48001122', 'c9b37b7c-17b3-11eb-a708-acde48001122', 'c9b37f96-17b3-11eb-a708-acde48001122', 'c9b3882e-17b3-11eb-a708-acde48001122', 'c9b38aea-17b3-11eb-a708-acde48001122', 'c9b38db0-17b3-11eb-a708-acde48001122', 'c9b3906c-17b3-11eb-a708-acde48001122', 'fa6614e4-c8c0-11ed-a172-0242ac130009', 'fa3e4e88-c8bf-11ed-bf64-0242ac130009', 'f85dab40-791d-11ee-88a6-0242ac130007', 'cb30d5aa-59c5-11ef-a45a-faffc210b420', '079e06e6-c8bf-11ed-a172-0242ac130009', '1d1bfbea-c8bf-11ed-bf64-0242ac130009', '24012b72-c8c1-11ed-a172-0242ac130009']}}], 'minimum_should_match': 1}} 
Exception detail: BadRequestError(400, 'search_phase_execution_exception', 'Text fields are not optimised for operations that require per-document field data like aggregations and sorting, so these operations are disabled by default. Please use a keyword field instead. Alternatively, set fielddata=true on [nodegroupid] in order to load field data by uninverting the inverted index. Note that this can use significant memory.')
Traceback (most recent call last):
  File "/Users/jwalls/prj/arches/arches/app/search/search.py", line 255, in search
    ret = self.es.search(**kwargs).body
          ~~~~~~~~~~~~~~^^^^^^^^^^
  File "/Users/jwalls/py313/lib/python3.13/site-packages/elasticsearch/_sync/client/utils.py", line 446, in wrapped
    return api(*args, **kwargs)
  File "/Users/jwalls/py313/lib/python3.13/site-packages/elasticsearch/_sync/client/__init__.py", line 4149, in search
    return self.perform_request(  # type: ignore[return-value]
           ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        "POST",
        ^^^^^^^
    ...<5 lines>...
        path_parts=__path_parts,
        ^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/Users/jwalls/py313/lib/python3.13/site-packages/elasticsearch/_sync/client/_base.py", line 271, in perform_request
    response = self._perform_request(
        method,
    ...<4 lines>...
        otel_span=otel_span,
    )
  File "/Users/jwalls/py313/lib/python3.13/site-packages/elasticsearch/_sync/client/_base.py", line 352, in _perform_request
    raise HTTP_EXCEPTIONS.get(meta.status, ApiError)(
        message=message, meta=meta, body=resp_body
    )
elasticsearch.BadRequestError: BadRequestError(400, 'search_phase_execution_exception', 'Text fields are not optimised for operations that require per-document field data like aggregations and sorting, so these operations are disabled by default. Please use a keyword field instead. Alternatively, set fielddata=true on [nodegroupid] in order to load field data by uninverting the inverted index. Note that this can use significant memory.')
2025-07-23 11:30:55,161 arches.app.search.search ERROR    2025-07-23 11:30:55.161339: WARNING: search failed for query: {'bool': {'should': [{'match_phrase_prefix': {'value': {'query': 'alpha gamma delta'}}}, {'match_phrase_prefix': {'value.folded': {'query': 'alpha gamma delta'}}}, {'match': {'value.folded': {'query': 'alpha gamma delta', 'fuzziness': 'AUTO', 'prefix_length': 3}}}], 'must': [{'term': {'language': 'en'}}], 'must_not': [], 'filter': [], 'minimum_should_match': 1}} 
Exception detail: BadRequestError(400, 'search_phase_execution_exception', 'Text fields are not optimised for operations that require per-document field data like aggregations and sorting, so these operations are disabled by default. Please use a keyword field instead. Alternatively, set fielddata=true on [top_concept] in order to load field data by uninverting the inverted index. Note that this can use significant memory.')
Traceback (most recent call last):
  File "/Users/jwalls/prj/arches/arches/app/search/search.py", line 255, in search
    ret = self.es.search(**kwargs).body
          ~~~~~~~~~~~~~~^^^^^^^^^^
  File "/Users/jwalls/py313/lib/python3.13/site-packages/elasticsearch/_sync/client/utils.py", line 446, in wrapped
    return api(*args, **kwargs)
  File "/Users/jwalls/py313/lib/python3.13/site-packages/elasticsearch/_sync/client/__init__.py", line 4149, in search
    return self.perform_request(  # type: ignore[return-value]
           ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        "POST",
        ^^^^^^^
    ...<5 lines>...
        path_parts=__path_parts,
        ^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/Users/jwalls/py313/lib/python3.13/site-packages/elasticsearch/_sync/client/_base.py", line 271, in perform_request
    response = self._perform_request(
        method,
    ...<4 lines>...
        otel_span=otel_span,
    )
  File "/Users/jwalls/py313/lib/python3.13/site-packages/elasticsearch/_sync/client/_base.py", line 352, in _perform_request
    raise HTTP_EXCEPTIONS.get(meta.status, ApiError)(
        message=message, meta=meta, body=resp_body
    )
elasticsearch.BadRequestError: BadRequestError(400, 'search_phase_execution_exception', 'Text fields are not optimised for operations that require per-document field data like aggregations and sorting, so these operations are disabled by default. Please use a keyword field instead. Alternatively, set fielddata=true on [top_concept] in order to load field data by uninverting the inverted index. Note that this can use significant memory.')
F2025-07-23 11:30:56,309 arches.app.search.search ERROR    2025-07-23 11:30:56.309646: WARNING: search failed for query: {'bool': {'should': [{'match_phrase_prefix': {'value': {'query': 'etiwanda'}}}, {'match_phrase_prefix': {'value.folded': {'query': 'etiwanda'}}}, {'match': {'value.folded': {'query': 'etiwanda', 'fuzziness': 'AUTO', 'prefix_length': 3}}}], 'must': [{'term': {'language': 'en'}}], 'must_not': [], 'filter': [{'terms': {'nodegroupid': ['307c4274-c8c1-11ed-a172-0242ac130009', '402c95f2-c8c1-11ed-87a7-0242ac130009', '4780cbb2-c8c0-11ed-a172-0242ac130009', '20000000-0000-0000-0000-100000000000', '6584b9c6-c8bf-11ed-a644-0242ac130009', '96f14fa6-c8bf-11ed-a644-0242ac130009', 'aa5a71de-c8c0-11ed-a172-0242ac130009', 'b74e29fe-c8c4-11ed-aab4-0242ac130009', 'be25bdf0-c8bf-11ed-a172-0242ac130009', 'bf611c24-c8be-11ed-b6a9-0242ac130009', 'd697b522-c8c0-11ed-87a7-0242ac130009', 'c9b3828e-17b3-11eb-a708-acde48001122', 'c9b37f96-17b3-11eb-a708-acde48001122', 'c9b38568-17b3-11eb-a708-acde48001122', 'c9b3882e-17b3-11eb-a708-acde48001122', 'c9b38aea-17b3-11eb-a708-acde48001122', 'c9b38db0-17b3-11eb-a708-acde48001122', 'c9b3906c-17b3-11eb-a708-acde48001122', 'fa6614e4-c8c0-11ed-a172-0242ac130009', 'fa3e4e88-c8bf-11ed-bf64-0242ac130009', 'f85dab40-791d-11ee-88a6-0242ac130007', 'cb30d5aa-59c5-11ef-a45a-faffc210b420', '079e06e6-c8bf-11ed-a172-0242ac130009', '1d1bfbea-c8bf-11ed-bf64-0242ac130009', '24012b72-c8c1-11ed-a172-0242ac130009']}}, {'terms': {'provisional': ['false']}}], 'minimum_should_match': 1}} 
Exception detail: BadRequestError(400, 'search_phase_execution_exception', 'Text fields are not optimised for operations that require per-document field data like aggregations and sorting, so these operations are disabled by default. Please use a keyword field instead. Alternatively, set fielddata=true on [nodegroupid] in order to load field data by uninverting the inverted index. Note that this can use significant memory.')
Traceback (most recent call last):
  File "/Users/jwalls/prj/arches/arches/app/search/search.py", line 255, in search
    ret = self.es.search(**kwargs).body
          ~~~~~~~~~~~~~~^^^^^^^^^^
  File "/Users/jwalls/py313/lib/python3.13/site-packages/elasticsearch/_sync/client/utils.py", line 446, in wrapped
    return api(*args, **kwargs)
  File "/Users/jwalls/py313/lib/python3.13/site-packages/elasticsearch/_sync/client/__init__.py", line 4149, in search
    return self.perform_request(  # type: ignore[return-value]
           ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        "POST",
        ^^^^^^^
    ...<5 lines>...
        path_parts=__path_parts,
        ^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/Users/jwalls/py313/lib/python3.13/site-packages/elasticsearch/_sync/client/_base.py", line 271, in perform_request
    response = self._perform_request(
        method,
    ...<4 lines>...
        otel_span=otel_span,
    )
  File "/Users/jwalls/py313/lib/python3.13/site-packages/elasticsearch/_sync/client/_base.py", line 352, in _perform_request
    raise HTTP_EXCEPTIONS.get(meta.status, ApiError)(
        message=message, meta=meta, body=resp_body
    )
elasticsearch.BadRequestError: BadRequestError(400, 'search_phase_execution_exception', 'Text fields are not optimised for operations that require per-document field data like aggregations and sorting, so these operations are disabled by default. Please use a keyword field instead. Alternatively, set fielddata=true on [nodegroupid] in order to load field data by uninverting the inverted index. Note that this can use significant memory.')
2025-07-23 11:30:56,319 arches.app.search.search ERROR    2025-07-23 11:30:56.319003: WARNING: search failed for query: {'bool': {'should': [{'match_phrase_prefix': {'value': {'query': 'etiwanda'}}}, {'match_phrase_prefix': {'value.folded': {'query': 'etiwanda'}}}, {'match': {'value.folded': {'query': 'etiwanda', 'fuzziness': 'AUTO', 'prefix_length': 3}}}], 'must': [{'term': {'language': 'en'}}], 'must_not': [], 'filter': [], 'minimum_should_match': 1}} 
Exception detail: BadRequestError(400, 'search_phase_execution_exception', 'Text fields are not optimised for operations that require per-document field data like aggregations and sorting, so these operations are disabled by default. Please use a keyword field instead. Alternatively, set fielddata=true on [top_concept] in order to load field data by uninverting the inverted index. Note that this can use significant memory.')
Traceback (most recent call last):
  File "/Users/jwalls/prj/arches/arches/app/search/search.py", line 255, in search
    ret = self.es.search(**kwargs).body
          ~~~~~~~~~~~~~~^^^^^^^^^^
  File "/Users/jwalls/py313/lib/python3.13/site-packages/elasticsearch/_sync/client/utils.py", line 446, in wrapped
    return api(*args, **kwargs)
  File "/Users/jwalls/py313/lib/python3.13/site-packages/elasticsearch/_sync/client/__init__.py", line 4149, in search
    return self.perform_request(  # type: ignore[return-value]
           ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        "POST",
        ^^^^^^^
    ...<5 lines>...
        path_parts=__path_parts,
        ^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/Users/jwalls/py313/lib/python3.13/site-packages/elasticsearch/_sync/client/_base.py", line 271, in perform_request
    response = self._perform_request(
        method,
    ...<4 lines>...
        otel_span=otel_span,
    )
  File "/Users/jwalls/py313/lib/python3.13/site-packages/elasticsearch/_sync/client/_base.py", line 352, in _perform_request
    raise HTTP_EXCEPTIONS.get(meta.status, ApiError)(
        message=message, meta=meta, body=resp_body
    )
elasticsearch.BadRequestError: BadRequestError(400, 'search_phase_execution_exception', 'Text fields are not optimised for operations that require per-document field data like aggregations and sorting, so these operations are disabled by default. Please use a keyword field instead. Alternatively, set fielddata=true on [top_concept] in order to load field data by uninverting the inverted index. Note that this can use significant memory.')
...............................................................................................................F...............................E..........................................................................................................................................
======================================================================
ERROR: setUpClass (tests.permissions.permission_tests.PermissionTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/jwalls/prj/arches/tests/base_test.py", line 93, in setUpClass
    super().setUpClass()
    ~~~~~~~~~~~~~~~~~~^^
  File "/Users/jwalls/py313/lib/python3.13/site-packages/django/test/testcases.py", line 1426, in setUpClass
    cls.setUpTestData()
    ~~~~~~~~~~~~~~~~~^^
  File "/Users/jwalls/prj/arches/tests/permissions/permission_tests.py", line 45, in setUpTestData
    cls.resource = Resource.objects.get(pk=cls.resource_instance_id)
                   ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jwalls/py313/lib/python3.13/site-packages/django/db/models/manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "/Users/jwalls/py313/lib/python3.13/site-packages/django/db/models/query.py", line 633, in get
    raise self.model.DoesNotExist(
        "%s matching query does not exist." % self.model._meta.object_name
    )
arches.app.models.resource.Resource.DoesNotExist: Resource matching query does not exist.

======================================================================
ERROR: setUpClass (tests.permissions.permissions_arches_default_allow_tests.ArchesDefaultAllowPermissionsTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/jwalls/prj/arches/tests/base_test.py", line 93, in setUpClass
    super().setUpClass()
    ~~~~~~~~~~~~~~~~~~^^
  File "/Users/jwalls/py313/lib/python3.13/site-packages/django/test/testcases.py", line 1426, in setUpClass
    cls.setUpTestData()
    ~~~~~~~~~~~~~~~~~^^
  File "/Users/jwalls/prj/arches/tests/permissions/base_permissions_framework_test.py", line 40, in setUpTestData
    cls.resource = Resource.objects.get(pk=cls.resource_instance_id)
                   ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jwalls/py313/lib/python3.13/site-packages/django/db/models/manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "/Users/jwalls/py313/lib/python3.13/site-packages/django/db/models/query.py", line 633, in get
    raise self.model.DoesNotExist(
        "%s matching query does not exist." % self.model._meta.object_name
    )
arches.app.models.resource.Resource.DoesNotExist: Resource matching query does not exist.

======================================================================
ERROR: setUpClass (tests.permissions.permissions_arches_default_deny_tests.ArchesDefaultDenyPermissionTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/jwalls/prj/arches/tests/base_test.py", line 93, in setUpClass
    super().setUpClass()
    ~~~~~~~~~~~~~~~~~~^^
  File "/Users/jwalls/py313/lib/python3.13/site-packages/django/test/testcases.py", line 1426, in setUpClass
    cls.setUpTestData()
    ~~~~~~~~~~~~~~~~~^^
  File "/Users/jwalls/prj/arches/tests/permissions/base_permissions_framework_test.py", line 40, in setUpTestData
    cls.resource = Resource.objects.get(pk=cls.resource_instance_id)
                   ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jwalls/py313/lib/python3.13/site-packages/django/db/models/manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "/Users/jwalls/py313/lib/python3.13/site-packages/django/db/models/query.py", line 633, in get
    raise self.model.DoesNotExist(
        "%s matching query does not exist." % self.model._meta.object_name
    )
arches.app.models.resource.Resource.DoesNotExist: Resource matching query does not exist.

======================================================================
ERROR: setUpClass (tests.views.resource_tests.ResourceViewTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/jwalls/prj/arches/tests/base_test.py", line 93, in setUpClass
    super().setUpClass()
    ~~~~~~~~~~~~~~~~~~^^
  File "/Users/jwalls/py313/lib/python3.13/site-packages/django/test/testcases.py", line 1426, in setUpClass
    cls.setUpTestData()
    ~~~~~~~~~~~~~~~~~^^
  File "/Users/jwalls/prj/arches/tests/views/resource_tests.py", line 65, in setUpTestData
    cls.resource = Resource.objects.get(pk=cls.resource_instance_id)
                   ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jwalls/py313/lib/python3.13/site-packages/django/db/models/manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "/Users/jwalls/py313/lib/python3.13/site-packages/django/db/models/query.py", line 633, in get
    raise self.model.DoesNotExist(
        "%s matching query does not exist." % self.model._meta.object_name
    )
arches.app.models.resource.Resource.DoesNotExist: Resource matching query does not exist.

======================================================================
FAIL: test_search_terms_permitted_user (tests.search.search_tests.SearchTests.test_search_terms_permitted_user)
Test search terms method with a permitted user
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/jwalls/prj/arches/tests/search/search_tests.py", line 231, in test_search_terms_permitted_user
    self.assertTrue("terms" in result and len(result["terms"]) == 1)
    ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AssertionError: False is not true

======================================================================
FAIL: test_load_package (tests.views.command_line_tests.CommandLineTests.test_load_package)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/jwalls/prj/arches/tests/views/command_line_tests.py", line 45, in test_load_package
    self.assertEqual(len(resources), 2)
    ~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^
AssertionError: 0 != 2

----------------------------------------------------------------------
Ran 560 tests in 277.755s

FAILED (failures=2, errors=4)
```

--- a/tests/test_settings.py
+++ b/tests/test_settings.py
@@ -122,3 +122,8 @@ if DOCKER:
         from arches.settings_docker import *
     except ImportError:
         pass
+
+
+# Tests shouldn't depend on celery running, so overwrite after settings_local import
+# This is enough to fool check_if_celery_available()
+CELERY_BROKER_URL = ""

